spec.bs

```diff
@@ -2953,13 +2993,16 @@ a {{ReportingBrowserSignals}} |browserSignals|, a [=direct from seller signals=]
   1. Let |reportFunctionName| be "`reportWin`".
   1. If |winner|'s [=generated bid/provided as additional bid=] is true:
     1. Set |reportFunctionName| be "`reportAdditionalBidWin`".
-  1. Let « ignored, |resultUrl|, |reportingBeaconMap|, |reportingMacroMap| » be the result of
-    [=evaluating a reporting script=] with |buyerReportingScript|, "`reportWin`", |leadingBidInfo|'s
-    [=leading bid info/auction config=]'s [=auction config/config idl=]'s
-    {{AuctionAdConfig/reportingTimeout}}, and
+  1. Let « ignored, |resultUrl|, |reportingBeaconMap|, |reportingMacroMap|, |paContributions| » be
+    the result of [=evaluating a reporting script=] with |buyerReportingScript|, "`reportWin`",

```

Comment:
just noticed that this "`reportWin`" needs to be |reportFunctinoName|, since it's "`reportAdditionalBidWin`" for additional bid.

Reply:
is fixed in https://github.com/WICG/turtledove/pull/1321/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR3001.